pkgs/by-name/ok/oku/package.nix

```diff
@@ -25,6 +25,12 @@ rustPlatform.buildRustPackage (finalAttrs: {
     hash = "sha256-utbey8DFXUWU6u2H2unNjCHE3/bwhPdrxAOApC+unGA=";
   };
 
+  # Avoiding "Illegal instruction (core dumped)" for binary caches
```

Comment:
```suggestion
  # Avoiding optimizations for reproducibility
```

Reply:
Thanks! Applied your suggestion in https://github.com/NixOS/nixpkgs/pull/423716/commits/234af4161bc52dc9328eb4208bec13fc4790d693